cfgov/v1/forms.py

```diff
@@ -160,6 +174,37 @@ def clean(self):
             self.data = data
         return self.cleaned_data
 
+    def set_interpreted_date_values(self, cleaned_data):
+        from_date = cleaned_data.get('from_date')
+        to_date = cleaned_data.get('to_date')
+        # if from_ or to_ is filled in, fill them both with sensible values
+        # if neither is filled in, leave them both blank
+        if from_date or to_date:
+            if from_date:
+                self.data['from_date'] = date.strftime(
+                    cleaned_data['from_date'], '%m/%d/%Y')
+            else:
+                # If there's a 'to_date' and no 'from_date',
+                #  use date of earliest possible filter result as 'from_date'
```

Comment:
Extra space here? Also, in comments overall I'd follow grammatical rules (capitalization and ending period) so it's clear when one is wrapping lines or not.